docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Router Deployment
+---
+
+import CliLogin from '/docs/_cli-login.md'
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md'
+
+This article is about deploying a Ziti router as a Linux system service. [This page](/learn/introduction/03-components.md#openziti-router) provides an overview of the architecture.
+
+Routers provide Ziti network entry and exit points for clients and smart routing through the mesh. You must have at least one router that is advertising an edge connection to clients. If you have more than one router, you must configure transport links between them to form the fabric mesh.
+
+## Installation
+
+The router package `openziti-router` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-router
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a configuration file and enrolling the router automatically. You will be prompted for the most relevant configuration values when installing interactively unless the answer is already known.
+
+Bootstrapping consists of these steps:
+
+1. Generate a one-shot configuration file from env answers and defaults.
+1. Enroll the router with the controller.
+
+At a minimum, you must set these to opt-in to bootstrapping.
+
+1. In `/opt/openziti/etc/router/service.env` (the answer file for the service)
+    1. Set `ZITI_BOOTSTRAP=true`
+1. In `/opt/openziti/etc/router/bootstrap.env` (the answer file for bootstrapping the router)
+    1. Set `ZITI_ENROLL_TOKEN` to the resulting token (JWT)
+
+Additionally, you probably want to set these:
+
+1. In `/opt/openziti/etc/router/bootstrap.env` (the answer file for bootstrapping the router)
+    1. Set `ZITI_CTRL_ADVERTISED_ADDRESS` to the FQDN of the controller (default: localhost).
+    1. Set `ZITI_ROUTER_ADVERTISED_ADDRESS` to the permanent FQDN of the router (default: localhost). This value can not be changed after enrollment.
+    1. Set `ZITI_ROUTER_MODE` (default: none) if this router's built-in tunneler will provide a proxy for services. Mode `tproxy` requires additional kernel capabilities in `/etc/systemd/system/ziti-router.service.d/override.conf` and DNS resolver configuration for the host.
+
+### Bootstrapping the Configuration
+
+The router's configuration is loaded from a YAML file ([reference](/reference/30-configuration/router.md)). The Linux system service will generate a valid configuration during the first startup, unless one already exists. You can modify the generated configuration in `/var/lib/ziti-router/config.yml`.
```

Comment:
same thoughts here as the controller. i think i like the idea that the configuration is generated from the answer file, which the process will then remove and putting a new answer file 'somewhere' is the indicator that you'd like to "reinstall" (or something, i'm not sold on the flow but it feels right to me at this time)... also it'd be worth calling out that when the router enrolls it will create a PKI which contains the server's advertised address in the server cert... which makes me think that we should indicate here that you basically only get ONE address for your router. Also alt server certs should be mentioned somewhere in the router page and why it's relevant and where/how and WS listeners (specifically for browzer)